src/main/java/gov/usgs/wma/waterdata/timeseries/TimeSeriesDao.java

```diff
@@ -30,6 +30,12 @@ public String getTimeSeries(String collectionId, String featureId, String timeSe
 		return getSqlSession().selectOne("groundwaterDailyValue.getGeoJson", params);
 	}
 
+	public String getTimeSeriesWaterML(String collectionId, String featureId, String timeSeriesId) {
+		Map<String,Object> params = collectionsParams.builder().collectionId(collectionId)
+		.featureId(featureId).timeSeriesId(timeSeriesId).build();
+		return getSqlSession().selectOne("groundwaterDailyValueWaterML.getWaterML", params);
+	}
+
```

Comment:
Javadocs would help here - I'm not sure why this is a selectOne (I'm sure it makes sense, I just can't recall how this matches up to the data)

Reply:
The selectOne() is used here because the sql returns just one row with one column. The column value is the WaterML xml
returned to the user.